cdap-proto/src/main/java/co/cask/cdap/proto/Notification.java

```diff
@@ -46,7 +46,8 @@
     TIME,
     STREAM_SIZE,
     PARTITION,
-    PROGRAM_STATUS
+    PROGRAM_STATUS,
+    HEART_BEAT
```

Comment:
think it's better to name it PROGRAM_HEART_BEAT, just in case we ever have other heartbeats.